website/tests/acceptance/sidebar-test.js

```diff
@@ -74,15 +78,23 @@ module('Acceptance | Sidebar filter', function (hooks) {
       .exists();
   });
 
+  test('should expand subsection when click a parent container', async function (assert) {
```

Comment:
question: I don't see any interactions in this, I assume it's more of a placeholder for the time being

Reply:
Yup, I added in the rest of the test now.